ArduinoSketch/ArduinoSketch.ino

```diff
@@ -51,12 +67,40 @@ void readInputFromRaspberryPi() {
     if (data == "play_off_sound") {
       playOffSound();
     }
+    if (data == "start_alarm") {
+      startAlarm();
+    }
+    if (data == "stop_alarm") {
+      stopAlarm();
+    }
+  }
+}
+
+bool shouldSoundAlarm() {
+  if (alarmLoopsCount >= 0) {
+    alarmLoopsCount = alarmLoopsCount + 1;
+
+    if (alarmLoopsCount > 60) {
+      // It's been 9 secondes (60 loops of 150ms (100ms @line 46 + 50ms @line 130))
```

Comment:
these 9 seconds MUST be lower than `REEMIT_AFTER_SECONDS` in `door-sensor.py`
They're here to prevent the alarm from being sound forever in case of malfunction/crash in `door-sensor.py`.